lapce-data/src/movement.rs

```diff
@@ -897,9 +897,13 @@ impl Movement {
 
 pub fn remove_n_at<T>(v: &mut Vec<T>, index: usize, n: usize) {
     match n.cmp(&1) {
-        Ordering::Equal => {v.remove(index);},
-        Ordering::Greater => {v.splice(index..index + n, std::iter::empty());},
-        _ => ()
+        Ordering::Equal => {
+            v.remove(index);
+        }
+        Ordering::Greater => {
+            v.drain(index..index + n);
```

Comment:
While this change is unrelated, rustfmt literally wouldn't let me save the file without making it. I could have chosen not to comit it, but I prefer well-formatted code.
And while I was at it, I replaced `splice` with `drain`, which it uses internally, so this code should be equivalent, but strictly simpler.